src/atomic_structure.hh

```diff
@@ -133,6 +133,27 @@ namespace rascal {
       this->positions = this->cell * scaled_positions;
     }
 
+    /**
+     * Wrap a set of explicitly provided positions into this structure's cell
+     *
+     * Like wrap(), but uses a set of user-specified positions instead of the
+     * atomic positions of this structure.j
+     *
+     * @param input_positions The positions to wrap
+     */
+    Positions_t wrap_explicit_positions(Positions_t input_positions) {
```

Comment:
Is there a test for this function then? I did not see one.

Reply:
If it comes to that, I'd rather delete it than write a bunch of test cases for this thing. It's exactly the same thing as `wrap()` only using user-supplied positions instead of the atomic ones.

Reply:
if it is not tested and not used, I would rather delete it yes